[EDGE] y2024 release notes updates (#2896)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-on-kubernetes-10-18-0-0.md

---
date: 2024-03-01
title: Cumulocity Edge on Kubernetes General Availability (GA)
change_type:
  - value: change-inv-3bw8e
    label: Announcement
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge on Kubernetes
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge-operator
version: 10.18.0.0
---
{{< product-c8y-iot >}} IoT Edge on Kubernetes has reached General Availability (GA) and is now ready for deployment in production environments.

> You can find the documentation for **{{< product-c8y-iot >}} Edge on Kubernetes** at https://cumulocity.com/docs/2024/edge-kubernetes/k8-edge-introduction/.

Reviewer: ```suggestion
Refer to the [{{< product-c8y-iot >}} Edge on Kubernetes](/edge-kubernetes/k8-edge-introduction/) documentation.
```
